homeassistant/components/onvif/camera.py

```diff
@@ -48,20 +48,30 @@
 ATTR_PAN = "pan"
 ATTR_TILT = "tilt"
 ATTR_ZOOM = "zoom"
+ATTR_DISTANCE = "distance"
+ATTR_SPEED = "speed"
+ATTR_MOVE_MODE = "move_mode"
+ATTR_CONTINUOUS_DURATION = "continuous_duration"
 
 DIR_UP = "UP"
 DIR_DOWN = "DOWN"
 DIR_LEFT = "LEFT"
 DIR_RIGHT = "RIGHT"
 ZOOM_OUT = "ZOOM_OUT"
 ZOOM_IN = "ZOOM_IN"
+
+CONTINUOUS_MOVE = "ContinuousMove"
+RELATIVE_MOVE = "RelativeMove"
+ABSOLUTE_MOVE = "AbsoluteMove"
 PTZ_NONE = "NONE"
```

Comment:
Is the value `"NONE"` used by the device library? Otherwise we can remove it and use `dict.get` to get the config item. Then the default value will be `None`.